__tests__/task.e2e.spec.js

```diff
@@ -0,0 +1,57 @@
+const request = require('supertest')
+const app = require('../src/app')
+const Task = require('../src/task')
+
+describe('Task', () => {
+  describe('POST /task/new', () => {
+    describe('when body is valid', () => {
+      it('should return CREATED status', async () => {
+        const response = await request(app)
+          .post('/task/new')
+          .set('Content-type', 'application/json')
+          .send({ description: 'desc', title: 'title' })
+
+        expect(response.status).toEqual(201)
+        expect(response.body.id).not.toBeNull()
+        expect(response.body.description).toEqual('desc')
+        expect(response.body.title).toEqual('title')
+      })
+    })
+
+    describe('when body is not valid or content type is not JSON application', () => {
+      it('should return BAD REQUEST status', async () => {
+        const response = await request(app).post('/task/new').send({})
+
+        expect(response.status).toEqual(400)
+      })
```

Comment:
🚲 shouldn't this be split into 2? One for an invalid body and another for the content type.